lib/truffle/rspec-workarounds.rb

```diff
@@ -0,0 +1,11 @@
+module RSpec
```

Comment:
`lib/truffle` seems very general, maybe it's time to have a `lib/truffle/workaround/` or so.